.gitlab-ci.yml

```diff
@@ -1255,6 +1255,34 @@ latest_release:
     - inv -e docker.publish --signed-pull --signed-push datadog/agent:${RELEASE}-jmx datadog/agent:latest-jmx
     - inv -e docker.publish --signed-pull --signed-push datadog/dogstatsd:${RELEASE} datadog/dogstatsd:latest
 
+#
+# Use this step to delete a tag of a given image
+# We call the Docker Hub API because docker cli doesn't support deleting tags
+# - remove the leading dot from the name
+# - set the IMAGE and TAG envvars
+# - in the gitlab pipeline view, trigger the step (in the first column)
+#
+.delete_docker_tag:
+  tags: [ "runner:docker", "size:large" ]
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker-notary:0.6.1
```

Comment:
Just one minor quibble: we shouldn't need Notary here

Reply:
@hkaj do we have an alternative image with aws access and without notary?

Reply:
yeah you can use 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker

Reply:
thanks, I'm going to test it

Reply:
@hkaj `/bin/bash: line 132: aws: command not found`

Reply:
hmm that's strange. Let's keep the notary one then if that's fine with you @trishankatdatadog and i'll have a look at what's in those images later.

Reply:
I've made more tests (with notary) and added `&>/dev/null` to the `inv` command to hide its output which dismissed you reviews. Can you re-approve please?